File: Serene/Serene.Script/Northwind/Order/FreightFormatter.cs

namespace Serene.Northwind
{
    using Serenity;
    using System.Runtime.CompilerServices;

    [Imported]
    public class FreightFormatter : ISlickFormatter
    {
        public string Format(SlickFormatterContext ctx)
        {
            return "<span class='freight-symbol'>" + 
                Q.HtmlEncode(ctx.Value) +
                "</span>";
        }
    }
}

File: Serene/Serene.Core/Modules/Northwind/Order/FreightFormatter.ts
namespace Serene.Northwind {

    @Serenity.Decorators.registerFormatter()
    export class FreightFormatter implements Slick.Formatter {
        format(ctx: Slick.FormatterContext) {
            return "<span class='freight-symbol'>" + Q.htmlEncode(ctx.value) + '</span>'
        }
    }
}